LandSandBoat/server
# Implement Food HP/MP mods (distinct from standard HP/MP mods)

**_I affirm:_**

## What does this pull request do?

I verified on retail that food HP/MP is both after Curse and before HPP/MPP

Adds FOOD_HP and FOOD_MP mods 

Apparently, it was needed to adjust CI so `scripts/items/m&p_dumpling.lua` was registering as a command (&) in bash because the script wasn't quoting changed files. Now it uses JSON which quotes them for us.

## Steps to test these changes

use food, see it work correctly in the formula

p.s. the way the food items were updated was with this command (I put it in the commit)


`find -name "*.lua" -exec bash -c 'grep "xi.effect.FOOD" "{}" && sed -i "s/xi.mod.MP,/xi.mod.FOOD_MP,/g ; s/xi.mod.HP,/xi.mod.FOOD_HP,/g" "{}"' \;`
